README.rst

```diff
@@ -48,6 +48,11 @@ user::
 
 .. warning:: This will destroy any previously stored data on your disk!
 
+f3probe assumes that higher bits of the drive block address are simply
+not connected in hardware. If you come across a fake drive that behaves
+differently, please open an issue reporting the details.
+
+
```

Comment:
This language is going to confuse lots of users. Most users just want to know if they have a fake drive or not, and what to do next. Helping them to do that is the way we fight flash fraud. It is better to add a section for people interested in the algorithm of `f3probe` and discuss technical aspects there.

Squash the next patch into this one since the next patch is just fixing this patch. You can use `git rebase -i master` at your branch.